models/get_topics.js

```diff
@@ -30,4 +30,15 @@ exports.selectArticleById = (article_id) => {
         }
         return rows[0];
       });
+}
+
+exports.selectAllArticles = () => {
+    const queryStr = `SELECT articles.author, title, articles.article_id, topic, articles.created_at, article_img_url, COUNT(comments.article_id) AS comment_count, SUM(comments.votes) AS votes FROM articles JOIN comments ON articles.article_id=comments.article_id GROUP BY articles.article_id ORDER BY articles.created_at;`
+    
+    return db.query(queryStr).then(({ rows }) => {
+        if(rows.length===0){
+            return Promise.reject({ status: 404, msg: "not found" });
+        }
```

Comment:
We shouldn't have a 404 here, even if there are no articles in the database, you would expect to get a 200 and an empty array